account_invoice_margin/report/account_invoice_report_view.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+    <record id="view_account_invoice_report_pivot" model="ir.ui.view">
```

Comment:
Why are you creating a totally new invoice analysis view instead of using existing one?